src/components/application_manager/src/commands/mobile/create_interaction_choice_set_request.cc

```diff
@@ -374,6 +376,7 @@ void CreateInteractionChoiceSetRequest::onTimeOut() {
   if (!error_from_hmi_) {
     SendResponse(false, mobile_apis::Result::GENERIC_ERROR);
   }
+  CommandRequestImpl::onTimeOut();
```

Comment:
@LuxoftAKutsan is this `onTimeOut()` call needed here? If `error_from_hmi_ = false` then SDL will send GENERIC_ERROR twice - from line 377 and then in `CommandRequestImpl::onTimeOut()`. Probably one of this calls should be removed